### PR TITLE
Expand docs on itry!

### PR DESCRIPTION
**Pull request by dbrgn**
More usage examples and link to modifiers docs.

### CODE REVIEW

**Comment by untitaker** [comment]
On mobile, if the link works im :+1:

**Comment by untitaker** [comment]
Thanks!

File: src/macros.rs [hunk 1]
```diff
@@ -22,8 +28,8 @@ macro_rules! itry {
     })
 }
 
-/// Unwrap the given Option or return a Ok(Response::new()) with the given modifier. The default
-/// modifier is status::BadRequest.
+/// Unwrap the given `Option` or return a `Ok(Response::new())` with the given
+/// modifier. The default modifier is `status::BadRequest.
```
Comment (SkylerLipthay) [inline src/macros.rs]: Missing a backtick here

Comment (untitaker) [inline src/macros.rs]: Jeez, sorry for merging :(

On 10 March 2016 11:58:14 CET, Skyler <notifications@github.com> wrote:
>> @@ -22,8 +28,8 @@ macro_rules! itry {
>>      })
>>  }
>>  
>> -/// Unwrap the given Option or return a Ok(Response::new()) with the
>given modifier. The default
>> -/// modifier is status::BadRequest.
>> +/// Unwrap the given `Option` or return a `Ok(Response::new())` with
>the given
>> +/// modifier. The default modifier is `status::BadRequest.
>
>Missing a backtick here
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/iron/iron/pull/435/files#r55663328

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by dbrgn** [comment]
It works locally :)

File: src/macros.rs (hunk 1 above)
Comment (dbrgn) [inline src/macros.rs]: Oops, I only checked `itry` docs in my browser, not `iexpect` :(

Do you want a PR, or will you fix it directly on master?

Comment (SkylerLipthay) [inline src/macros.rs]: how dare you :stuck_out_tongue_winking_eye:

Comment (untitaker) [inline src/macros.rs]: Fixed https://github.com/iron/iron/commit/44390ca93d7a4df93e7fd8397aadb6d84483f232

**Pull request merged**